platform/ios/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed a crash or console spew when MGLMapView is initialized with a frame smaller than 64 points wide by 64 points tall. ([#8562](https://github.com/mapbox/mapbox-gl-native/pull/8562))
 * The error passed into `-[MGLMapViewDelegate mapViewDidFailLoadingMap:withError:]` now includes a more specific description and failure reason. ([#8418](https://github.com/mapbox/mapbox-gl-native/pull/8418))
 * Fixed an issue where gesture recognizers associated with map view interactivity were not disabled when their related interactions were disabled. ([#8304](https://github.com/mapbox/mapbox-gl-native/pull/8304))
+* Added more Swift integration test coverage to help ensure that the effects of subtle SDK changes to Swift bridging are understood and captured in future release notes. ([#8729](https://github.com/mapbox/mapbox-gl-native/pull/8729))
```

Comment:
This is an internal change to our testing infrastructure; not sure we need a changelog entry about it.